launch/world_modeling/people_recognition.launch

```diff
@@ -23,6 +23,8 @@
     <arg name="keras_face_model" default="~/data/keras_models/age_gender/weights.28-3.73.hdf5" />
     <arg name="openpose_models" default="~/openpose/models/" />
 
+    <arg name="config_file" default="$(find people_recognition_3d)/config/config.yaml" />
```

Comment:
The config is for `people_recognition_3d`. As multiple nodes are launched in this file, I think this should be represented in the name.